docs/aws/aws.md

```diff
@@ -103,29 +104,30 @@ repeat the verification process for each subnet ID.
     
        ```shell        
        ./osd-network-verifier egress \
-           --subnet-id <subnet_id> 
+           --subnet-id <subnet_id> --profile <aws-profile-name>
         ```
    
         Additional optional flags for overriding defaults:
       ```shell
-      --cloud-tags stringToString   (optional) comma-seperated list of tags to assign to cloud resources (default [osd-network-verifier=owned,red-hat-managed=true,Name=osd-network-verifier])
+      --cloud-tags stringToString   (optional) comma-seperated list of tags to assign to cloud resources e.g. --cloud-tags key1=value1,key2=value2 (default [osd-network-verifier=owned,red-hat-managed=true,Name=osd-network-verifier])
       --debug                       (optional) if true, enable additional debug-level logging
-      --image-id string             (optional) specify cloud image for the compute instance
-      --instance-type string        compute instance type (default "(optional) t3.micro")
+      --image-id string             (optional) cloud image for the compute instance
+      --instance-type string        (optional) compute instance type (default "t3.micro")
       --kms-key-id string           (optional) ID of KMS key used to encrypt root volumes of compute instances. Defaults to cloud account default key
-      --region string               (optional) compute instance region. If absent, environment var AWS_DEFAULT_REGION will be used, if set (default "us-east-1")
+      --region string               (optional) compute instance region. If absent, environment var AWS_REGION will be used, if set (default "us-east-2")
+      --profile string              (optional) AWS profile. If present, any credentials passed with CLI will be ignored.
       --subnet-id string            source subnet ID
       --timeout duration            (optional) timeout for individual egress verification requests (default 2s). If timeout is less than 2s, it would likely cause false negatives test results.
-       ```
+         ```
+   
        Get cli help:
     
         ```shell
         ./osd-network-verifier egress --help
         ```
 
-##### 1.1.2 Go implementation Examples #####
-- [AWS Go SDK v1](../../examples/aws/verify_egressv1.go)  
```

Comment:
There's no need to pass version specific creds object to client any more. Downstream was previously creating version specific cred object, but will now be providing either version agnostic profile or AWS keys to cloudclient.